docs/existing_data.rst

```diff
@@ -17,8 +17,10 @@ A Patroni cluster can be started with a data directory from a single-node Postgr
 
 .. code-block:: sql
 
-   CREATE USER $PATRONI_SUPERUSER_USERNAME WITH SUPERUSER ENCRYPTED PASSWORD '$PATRONI_SUPERUSER_PASSWORD';
-   CREATE USER $PATRONI_REPLICATION_USERNAME WITH REPLICATION ENCRYPTED PASSWORD '$PATRONI_REPLICATION_PASSWORD';
+   CREATE USER PATRONI_SUPERUSER_USERNAME WITH SUPERUSER ENCRYPTED PASSWORD 'PATRONI_SUPERUSER_PASSWORD';
+   CREATE USER PATRONI_REPLICATION_USERNAME WITH REPLICATION ENCRYPTED PASSWORD 'PATRONI_REPLICATION_PASSWORD';
```

Comment:
Hmm. I would prefer to leave `$` char as it is.

Reply:
The reason I changed it is because the sphinx autodoc complains that this isn't a valid SQL command. I don't think those `$` symbols are valid and thought perhaps would confuse the user.

I can revert but I also added something to the tox.ini so that a doc build would fail on any warnings. If I revert I'll have to turn that off. That's not a massive problem, it would mean we'd lose the early warning on doc syntax problems, but we can lint docs with other things, e.g. pydocstyle. I was wondering whether to suggest adding that so we have doc linting. There is a tool called prospector which I've found to be very comprehensive with lint checks. That would at least allow us to add exceptions for specific errors we want to ignore (like this). However, I don't think it has this language inspection like the original method in sphinx.